layouts/community/ortho_4x12/ajp10304/keymap.c

```diff
@@ -1,41 +1,4 @@
-#include "planck.h"
-#include "action_layer.h"
-#include "eeconfig.h"
-#include "keymap_uk.h"
-
-extern keymap_config_t keymap_config;
-
-enum planck_layers {
-  _QWERTY,
-  _MAC,
-  _LOWER,
-  _MLWR,
-  _RAISE,
-  _MRSE,
-  _FUNC,
-  _MFNC,
-  _FUNC2,
-  _MFNC2,
-  _ADJUST,
-  _MOUSE
-};
-
-enum planck_keycodes {
-  QWERTY = SAFE_RANGE,
-  MAC,
-  FUNC,
-  MFNC,
-  FUNC2,
-  MFNC2,
-  LOWER,
-  MLWR,
-  RAISE,
-  MRSE,
-  MOUSE,
-  DYNAMIC_MACRO_RANGE
-};
-
-#include "dynamic_macro.h"
+#include "ajp10304.h"
```

Comment:
```suggestion
#include "ajp10304.h"
#include "keymap_uk.h"
```

Reply:
Eh?  You told me to remove keymap_uk.h above and now you want it back in?

Reply:
Nothing defined in `keymap_uk.h` is used in `ajp10304.h`, although it is implicitly included in this file by the inclusion of the latter. This just moves it closer to where it's actually used.

Reply:
Ok, done.